### DISCUSSION TITLE
Join Hitchwiki

### DISCUSSION THREAD

**Discussion by tillwenke** [Hitchwiki/hitchwiki#199] [Announcements]
Just add a message below to get access to our repositories and contribute :)
state:
- which way you'd love to contribute
- your [Hitchwiki user name](https://hitchwiki.org/en/index.php?title=Special:UserLogin&returnto=Special%3ARequestAccount) 

@Hitchwiki/admins  will take care of you.

**Comment by nikonotfound** [comment]
@NikoOnTheRoad

Discuss if there's a topic coming up.

**Comment by Atopilak16** [comment]
First timer here attempted to head west from Kanata.

**Reply by tillwenke** [reply]
nice. did you request a https://hitchwiki.org account already?
is there anything particular you are interested here?

**Reply by Atopilak16** [reply]
Just request one. Thank u

**Comment by GlassUnagi** [comment]
Hello!

I'm currently going from France to Georgia by hitchhiking, right now I'm in Turkey. I would like to thanks hitchwiki by contributing with the section hitchhiking out of cities, update some spots and add some cities if so. 

@Eel

**Reply by tillwenke** [reply]
Did you get an account on hitchwiki already? if so you should be able to edit wiki pages there

**Reply by leelee85527** [reply]
No I don't how do I request one

**Reply by leon-wbr** [reply]
It seems the account request page is broken right now, usually the link at the top right of the site should work: https://hitchwiki.org/en/Special:RequestAccount

I'll let you know when we fixed it.

**Reply by leon-wbr** [reply]
I believe the link should be working again. Let me know if not.
